src/vkscript/executor_test.cc

```diff
@@ -195,10 +195,11 @@ class EngineStub : public Engine {
     return {};
   }
 
-  Result DoProcessCommands(uint32_t*,
-                           uint32_t*,
-                           uint32_t*,
-                           const void**) override {
+  Result DoProcessCommands() override { return {}; }
```

Comment:
The change of DoProcessCommand into 3 methods could also possibly be split out to another CL